docs/src/pages/lab/slider/slider.md

```diff
@@ -27,6 +27,10 @@ Sliders reflect the current state of the settings they control.
 
 {{"demo": "pages/lab/slider/StepSlider.js"}}
 
+## Slider with steps with edge values
+
```

Comment:
The wording sounds weird to me. I think this can be included in `StepSlider` since it's the default behavior and not configurable.